thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/template.hbs

```diff
@@ -198,6 +198,7 @@
             <tr class="te-anomaly-table__row">
                <td class="te-anomaly-table__cell">
                 <ul class="te-anomaly-table__list">
+                  <li class="te-anomaly-table__list--shadow">{{anomaly.index}}</li>
```

Comment:
Looks like a misuse of BEM here, `te-anomaly-table__list` and its modifiers `te-anomaly-table__list--shadow` should be on the same element
https://medium.com/fed-or-dead/battling-bem-5-common-problems-and-how-to-avoid-them-5bbd23dee319

Reply:
yes - this is something I'm gradually cleaning up from early implementations. This case is resolved.